src/views/v2/components/read-more.php

```diff
@@ -9,15 +9,14 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.10
+ * @version TBD
  *
  */
```

Comment:
* :no_entry_sign: There must be exactly one blank line after the file comment
* :no_entry_sign: Missing @package tag in file comment
* :no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 8c2446a4f511b5c6f79df7515a6d10db</sub>